XamlControlsGallery/ControlPages/TextBoxPage.xaml

```diff
@@ -34,14 +34,14 @@
         </local:ControlExample>
         <local:ControlExample x:Name="Example3" HeaderText="A read-only TextBox with various properties set.">
             <TextBox Text="I am super excited to be here!" IsReadOnly="True" FontFamily="Arial" FontSize="24"
-                     FontStyle="Italic" CharacterSpacing="200" Foreground="CornflowerBlue" 
+                     FontStyle="Italic" CharacterSpacing="200" Foreground="#669DFF" 
```

Comment:
It seems that this color does not have sufficient contrast in light theme: 
![Screenshot if contrast ratio between light theme background color and the color #669DFF](https://user-images.githubusercontent.com/16122379/153484032-0c1d5242-70d3-4d25-bf2b-3ab911c93fa5.png)

Reply:
Good catch Marcel! Thanks for the feedback. Do you know what color would work for both light and dark themes? I tried to find one that worked for both but I don't think that it is possible. Wondering if there is a way to change the color based on the theme. Let me know what you think.

Reply:
I am not entirely sure if there is a color that would have a contrast ratio of 4.5 to both the light and dark background. Alternatively, we could use a theme resource that would change colors based on theme. If you like, I can also dig deeper and try to find a color that could fit the criteria.

Reply:
One thing that just came to mind is if we actually need to meet the 4.5 contrast ratio of if the 3.0 ratio is enough since we have a large font size here.

Reply:
That's a good point. In this case I believe that #5178BE should be good for both light and dark modes.
![image](https://user-images.githubusercontent.com/90726278/153521604-76a89c3d-8347-43f0-b81e-e3ba8976f332.png)
![image](https://user-images.githubusercontent.com/90726278/153521632-4b2c00b9-cc6f-4e4a-b473-c43c73f615c0.png)


Let me know what you think :)

Reply:
That colors looks good to me!

Reply:
Just updated the color code :)

Reply:
Oh right, guess I was a bit to eager to approve the PR. LGTM to me now :)